components/CustomerNavigation/CustomerNavigation.js

```diff
@@ -0,0 +1,24 @@
+import { Container } from "../Buttons/styles";
+import { EditButton } from "../CustomerInfo/styles";
+
+function getGoogleMapsLink(customer) {
+  const { street, location, plz } = customer;
+  const address = `${street}, ${location}, ${plz}`;
```

Comment:
```suggestion
  const address = `${street}, ${location}, ${plz}`;
```
"plz" should probably be areaCode 🤔

Reply:
Im sorry, but i dont get whats the change in this suggestion? 

Also i dont know what areaCode means :see_no_evil: